src/analyzer/analyzerqueue.cpp

```diff
@@ -457,7 +457,7 @@ AnalyzerQueue* AnalyzerQueue::createAnalysisFeatureAnalyzerQueue(
         UserSettingsPointer pConfig, TrackCollection* pTrackCollection) {
     AnalyzerQueue* ret = new AnalyzerQueue(pTrackCollection);
 
-    if (pConfig->getValue<bool>(ConfigKey("[Library]", "GenerateWaveformsWithAnalysis"))) {
+    if (pConfig->getValue<bool>(ConfigKey("[Library]", "EnableWaveformGenerationWithAnalysis"))) {
```

Comment:
This is a very long name, but it is clear. Anyone have a suggestion for a shorter name that is still clear?